### ISSUE TITLE
FR: Wrap JS-Repl live feedback if Word Wrap is enabled. Possible?

### ISSUE THREAD

**Issue by VKondakoff** [axilleasiv/vscode-javascript-repl-docs#9]
Maybe it is a good idea to wrap JS-Repl feedback if Word Wrap is currently enabled?

<img width="973" alt="Image on 2019-11-14 04 42 28 PM" src="https://user-images.githubusercontent.com/230824/68862012-c16f2280-06fd-11ea-8059-341e6a9c8b26.png">

Another one consideration: currently the JS-Repl feedback is cropped around 90th character. Can this be configurable?

**Comment by axilleasiv** [comment]
Yes, the Repl feedback is cropped on the 80th character. At this moment, this is not configurable, but this can be configurable with a default value for example 80 that it works now.

I agree with you that when feedback results are cropped some sign like three dots is necessary for users.

As for word wrap consideration when is enabled, I am not sure about it but I find this interesting, so I will try to implement this as a feature and if there are not any problems with the line numbers of feedback, I will include this to a next release.

Thank you, @VKondakoff
